src/sql/orig_results.sql

```diff
@@ -0,0 +1,15 @@
+CREATE VIEW total_counts AS
```

Comment:
frequency distribution of tokens?

Reply:
correct, this query give us the same results we reported in the preliminary results report.